undefined

```diff
@@ -0,0 +1,13 @@
+length = int(input())
+n = int(input())
+
+for _ in range(n):
+    w, h = map(int, input().split())
+    
+    if w < length or h < length:
```

Comment:
The condition in line 7 should use 'and' instead of 'or'. It should be 'if w < length and h < length:'